core/src/main/resources/db/changelog.xml

```diff
@@ -14,4 +14,27 @@
     <!-- initial dump for new installations -->
     <include file="changelog_base.xml" relativeToChangelogFile="true" />
 
+    <changeSet author="OliverKlette85" context="new" id="20151229220700">
+        <createTable tableName="user">
+            <column name="id" type="VARCHAR(255)">
```

Comment:
here it is a varchar in your model an int. ;) change to uuid and reduce the size.

!fix